nltk_contrib/timex3.py

```diff
@@ -1,19 +1,15 @@
 # Code for tagging temporal expressions in text
 # For details of the TIMEX format, see http://timex2.mitre.org/
+# Converted to Python3 by Brian Hockenmaier in 2019
 
 import re
 import string
 import os
 import sys
+from datetime import datetime, timedelta
```

Comment:
The new timedelta features of python3 allow us to remove the dependency on mx.DateTime